src/components/pages/vision/introduction.tsx

```diff
@@ -52,20 +67,42 @@ const Introduction: React.FC = () => {
           Vision
         </h1>
       </div>
-      <div className="w-full mx-auto md:max-w-md">
+      <div className="w-full ">
         {sectionCopy.map((copy, i) => {
           return (
-            <div key={i} className="mt-20 sm:mt-30">
-              <h2 className="mb-10 text-3xl font-bold sm:text-4xl sm:mb-14">
-                {copy.title}
-              </h2>
-              {copy.descriptions.map((description, index) => {
-                return (
-                  <p className="mt-8" key={index}>
-                    {description}
-                  </p>
-                )
-              })}
+            <div
+              key={i}
+              className="grid grid-cols-1 gap-8 mt-20 sm:mt-40 sm:gap-20 md:grid-cols-2"
+            >
+              <div className={i === 1 ? 'sm:order-last' : ''}>
+                <h2 className="mb-10 text-3xl font-bold sm:text-4xl sm:mb-14">
+                  {copy.title}
+                </h2>
+                {copy.descriptions.map((description, index) => {
+                  return (
+                    <p className="mt-8 text-xl" key={index}>
+                      {description}
+                    </p>
+                  )
+                })}
+              </div>
+
+              <div className={i === 1 ? '' : 'flex sm:justify-end'}>
+                <div>
+                  <div className="max-w-sm pb-10 pt-36 vision-quote">
+                    <p className="text-xl px-15">{copy.quote}</p>
+                  </div>
+                  <div className="mt-10 ml-10">
+                    <img
+                      className="float-left pr-5"
+                      src={copy.quoteImage}
+                      alt={copy.quoteAuthor}
+                    />
+                    <p className="pt-2">{copy.quoteAuthor},</p>
+                    <p className="">{copy.quoteAuthorTitle}</p>
+                  </div>
```

Comment:
Try to avoid using float. I recommend using flex here.
```suggestion
                  <div className="mt-10 ml-10 flex">
                    <img
                      className="pr-5"
                      src={copy.quoteImage}
                      alt={copy.quoteAuthor}
                    />
                    <div>
                      <p className="pt-2">{copy.quoteAuthor},</p>
                      <p className="">{copy.quoteAuthorTitle}</p>
                    </div>
                  </div>
```

Reply:
Changed to flex and combined author info